easybuild/easyconfigs/l/libbacktrace/libbacktrace-20240505-GCC-12.3.0.eb

```diff
@@ -0,0 +1,50 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = 'libbacktrace'
+version = '20240505'
+local_commit = '11427f3'
+
+homepage = 'https://github.com/ianlancetaylor/libbacktrace/tree/master'
+description = """
+A C library that may be linked into a C/C++ program to produce symbolic backtraces
+Initially written by Ian Lance Taylor iant@golang.org.
+This is version 1.0. It is likely that this will always be version 1.0."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+sources = [{
+    'filename': SOURCE_TAR_GZ,
```

Comment:
Change to SOURCE_TAR_XZ and we can get persistenty checksums, so update checksum to after taht